src/avm1/lib/AVM1MovieClip.ts

```diff
@@ -343,7 +343,7 @@ module Shumway.AVM1.Lib {
       var nativeObject = this.as3Object;
       var maxDepth = 0;
       for (var i = 0, numChildren = nativeObject.numChildren; i < numChildren; i++) {
-        var child = nativeObject._lookupChildByIndex(i);
+        var child = nativeObject._children[i];
```

Comment:
I'm okay with that, but I would rather add a LookupChildOptions parameter to the _lookupChildByIndex (see _lookupChildByName)